test/puppetlabs/puppetdb/testutils/db.clj

```diff
@@ -63,9 +63,8 @@
       (assert (valid-sql-id? name))
       name)))
 
-(defn init-db [db read-only?]
-  (jdbc/with-db-connection db (migrate! db))
-  (jdbc/pooled-datasource (assoc db :read-only? read-only?)))
```

Comment:
Why did we need to remove the pooled-datasource here? I don't really know the context behind what it's doing and if it's ok to remove.